src/Entity.php

```diff
@@ -1658,48 +1658,25 @@ public static function showInventoryOptions(Entity $entity)
-            $result = null;
-            if ($entity->fields[$field] == self::CONFIG_PARENT) {
-                if ($strategy_field === null) {
-                    $strategy_field = $field;
-                }
-                $inherited_strategy = self::getUsedConfig($strategy_field, $entity->fields['entities_id']);
-                $inherited_value    = $inherited_strategy === 0
-                    ? self::getUsedConfig($strategy_field, $entity->fields['entities_id'], $field)
-                    : $inherited_strategy;
-                $result = self::inheritedValue(
-                    self::getSpecificValueToDisplay($field, $inherited_value),
-                    false,
-                    false
-                );
-            }
-            return $result;
-        };
-
         $inheritance_labels = [
-            'entities_id_software'          => $fn_get_inheritance_label('entities_id_software', 'entities_strategy_software'),
-            'transfers_id'                  => $fn_get_inheritance_label('transfers_id', 'transfers_strategy'),
-            'autofill_buy_date'             => $fn_get_inheritance_label('autofill_buy_date'),
-            'autofill_order_date'           => $fn_get_inheritance_label('autofill_order_date'),
-            'autofill_delivery_date'        => $fn_get_inheritance_label('autofill_delivery_date'),
-            'autofill_use_date'             => $fn_get_inheritance_label('autofill_use_date'),
-            'autofill_warranty_date'        => $fn_get_inheritance_label('autofill_warranty_date'),
-            'autofill_decommission_date'    => $fn_get_inheritance_label('autofill_decommission_date'),
-            'agent_base_url'                => self::inheritedValue(
-                self::getUsedConfig('agent_base_url', $ID, '', ''),
-                false,
-                false
-            ),
+            'entities_id_software'          => $entity->getInheritedValueBadge('entities_id_software', 'entities_strategy_software'),
+            'transfers_id'                  => $entity->getInheritedValueBadge('transfers_id', 'transfers_strategy'),
+            'autofill_buy_date'             => $entity->getInheritedValueBadge('autofill_buy_date'),
+            'autofill_order_date'           => $entity->getInheritedValueBadge('autofill_order_date'),
+            'autofill_delivery_date'        => $entity->getInheritedValueBadge('autofill_delivery_date'),
+            'autofill_use_date'             => $entity->getInheritedValueBadge('autofill_use_date'),
+            'autofill_warranty_date'        => $entity->getInheritedValueBadge('autofill_warranty_date'),
+            'autofill_decommission_date'    => $entity->getInheritedValueBadge('autofill_decommission_date'),
+            'agent_base_url'                => $entity->getInheritedValueBadge(field: 'agent_base_url', inherit_parent_value: null),
```

Comment:
Inherited value should not be shown on root entity.
![image](https://github.com/glpi-project/glpi/assets/33253653/18408285-637e-458b-9f24-fd49be1f2a88)

Reply:
Is there something different in my DB? This field is just empty/null by default and AFAIK, an empty value is supposed to be treated as inherited by parent.

Reply:
My screenshot was taken in the root entity context. The root entity cannot inherit anything.